--- origsrc/ggz-server-0.0.14.1/game_servers/ggzcards/ai.c	2008-01-05 13:03:00.000000000 -0600
+++ src/ggz-server-0.0.14.1/game_servers/ggzcards/ai.c	2008-06-05 16:33:02.546875000 -0500
@@ -40,7 +40,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <unistd.h>
-#include <wait.h>
+#include <sys/wait.h>
 
 #include "ai.h"
 #include "common.h"
--- origsrc/ggz-server-0.0.14.1/game_servers/widelands/widelands_server.cpp	2008-01-05 13:03:02.000000000 -0600
+++ src/ggz-server-0.0.14.1/game_servers/widelands/widelands_server.cpp	2008-06-05 16:36:25.125000000 -0500
@@ -122,6 +122,7 @@
 			addr = (struct sockaddr*)malloc(addrsize);
 			ret = getpeername(channel, addr, &addrsize);
 
+#ifdef AF_INET6
 			// FIXME: IPv4 compatibility?
 			if(addr->sa_family == AF_INET6)
 			{
@@ -129,7 +130,9 @@
 				inet_ntop(AF_INET6, (void*)&(((struct sockaddr_in6*)addr)->sin6_addr),
 					ip, INET6_ADDRSTRLEN);
 			}
-			else if(addr->sa_family == AF_INET)
+			else
+#endif
+			if(addr->sa_family == AF_INET)
 			{
 				ip = (char*)ggz_malloc(INET_ADDRSTRLEN);
 				inet_ntop(AF_INET, (void*)&(((struct sockaddr_in*)addr)->sin_addr),
--- origsrc/ggz-server-0.0.14.1/ggzd/ggzd.conf.example.in	2008-01-05 13:03:09.000000000 -0600
+++ src/ggz-server-0.0.14.1/ggzd/ggzd.conf.example.in	2008-06-05 20:53:47.843750000 -0500
@@ -58,8 +58,8 @@
 # Transport Layer Security options
 #EncryptionUse = 0
 #EncryptionPassword = ggzrocksmore
-#EncryptionKey = @prefix@/etc/ggzsite-example.pem
-#EncryptionCert = @prefix@/etc/ggzsite-example.pem
+#EncryptionKey = @sysconfdir@/ggzsite-example.pem
+#EncryptionCert = @sysconfdir@/ggzsite-example.pem
 
 # Announcements (none by default), metaserver must point to a GGZ metaserver
 #AnnounceLAN = 1
@@ -165,7 +165,7 @@
 
 # Configuration directory
 #   This is where the server looks for game and room description files
-#ConfDir = @prefix@/etc/ggzd
+#ConfDir = @sysconfdir@/ggzd
 
 # Data directory
 #   Where all server tables are stored if SQL is not used
--- origsrc/ggz-server-0.0.14.1/ggzdmod/Makefile.am	2008-01-05 13:03:05.000000000 -0600
+++ src/ggz-server-0.0.14.1/ggzdmod/Makefile.am	2008-06-05 15:34:13.109375000 -0500
@@ -9,7 +9,7 @@
 noinst_LTLIBRARIES = libggzdmod.la
 endif
 
-libggzdmod_la_LDFLAGS = $(LIBGGZ_LDFLAGS) -version-info 6:0:0
+libggzdmod_la_LDFLAGS = $(LIBGGZ_LDFLAGS) -version-info 6:0:0 -no-undefined
 libggzdmod_la_LIBADD = $(LIB_GGZ)
 libggzdmod_la_SOURCES = ggzdmod.h ggzdmod-ggz.h \
 			ggzdmod.c \
--- origsrc/ggz-server-0.0.14.1/ggzdmod++/Makefile.am	2008-01-05 13:02:53.000000000 -0600
+++ src/ggz-server-0.0.14.1/ggzdmod++/Makefile.am	2008-06-05 15:34:04.515625000 -0500
@@ -1,6 +1,6 @@
 lib_LTLIBRARIES = libggzdmod++.la
 libggzdmod___la_SOURCES = ggzgameserver.cpp net.cpp mnet.cpp
-libggzdmod___la_LDFLAGS = $(LIBGGZ_LDFLAGS) -version-info 1:0:0
+libggzdmod___la_LDFLAGS = $(LIBGGZ_LDFLAGS) -version-info 1:0:0 -no-undefined
 libggzdmod___la_LIBADD = $(top_builddir)/ggzdmod/libggzdmod.la $(LIB_GGZ)
 libggzdmod___la_CPPFLAGS = -I $(srcdir) -I $(top_srcdir)/ggzdmod $(LIBGGZ_INCLUDES)
 
